comp/logs/auditor/impl/auditor.go

```diff
@@ -253,6 +282,10 @@ func (a *registryAuditor) cleanupRegistry() {
 	expireBefore := time.Now().UTC().Add(-a.entryTTL)
 	for path, entry := range a.registry {
 		if entry.LastUpdated.Before(expireBefore) {
+			if a.tailedSources[path] {
```

Comment:
question: Would it be more clear to this reverse this conditional? Something like
if !a.tailedSources[path] {
    a.log.Debug("TTL"...)
    delete()
} else {
    a.log.Debugf("TTL for...")
}

Both ways work, I'm just wondering if it would be more clear to have the conditional be something like "if (condition) then do work" as opposed to "if condition, skip". I can see either way though